client/extensions/woocommerce/index.js

```diff
@@ -144,8 +145,9 @@ function getStoreSidebarItemButtons() {
 
 function addStorePage( storePage, storeNavigation ) {
 	page( storePage.path, siteSelection, storeNavigation, function( context ) {
+		const component = React.createElement( storePage.container, { params: context.params } );
 		renderWithReduxStore(
-			React.createElement( storePage.container, { className: 'woocommerce', params: context.params } ),
+			React.createElement( App, { component } ),
```

Comment:
Why not just use children instead of defining an extra parameter for the component? It does look like you're doing anything special with this component anyway.